modules/Bio/EnsEMBL/Variation/Pipeline/CheckTranscriptVariation.pm

```diff
@@ -53,6 +53,27 @@ sub run {
     ## Update meta table with this analysis
     $self->update_meta();
 
+    my $species = $self->param('species');
+    my $adaptor = $self->get_adaptor($species, 'variation');
+    my $dbname = $adaptor->dbc()->dbname();
+
+    # prepare old_server_uri to which to compare against
+    my $old_server_uri = $self->param('old_server_uri');
+
+    unless ($old_server_uri) {
+      my $user    = $adaptor->dbc()->user;
+      my $port    = $adaptor->dbc()->port;
+      my $host    = $adaptor->dbc()->host;
+      my $pass    = $adaptor->dbc()->pass;
+      my $release = $self->param('ensembl_release') - 1;
+      $old_server_uri ||= sprintf("mysql://%s:%s@%s:%s/%s",
+                                $user, $pass, $host, $port, $release);
+    }
+
+    $self->param('dc_output_ids', { name => $dbname, old_server_uri => [ $old_server_uri ] });
```

Comment:
It should be `dbname` instead of `name` - 
```suggestion
    $self->param('dc_output_ids', { dbname => $dbname, old_server_uri => [ $old_server_uri ] });
```

Reply:
That was the problem! Now it works.